bower.json

```diff
@@ -20,7 +20,8 @@
     "angular-sanitize": "~1.3.4",
     "uri.js": "~1.14.2",
     "angular-busy": "~4.1.2",
-    "prism": "f658c05"
+    "prism": "f658c05",
+    "angular-value-format": "~0.1.0"
```

Comment:
I see a clear benefit of separating such functionality. Re-usability and all. What bothers me is the number of my PRs hanging for half a year in other projects we depend on. This module is simple enough so it would not require a lot of changes, though, I have to ask, can you guarantee you are going to support it for at least a couple of years? Because if not, I would rather prefer we would make this a part of st2web from day one.